### ISSUE TITLE
dollar-variable-no-missing-interpolation rule doesn't work with custom properties

### ISSUE THREAD

**Issue by in-in** [kristerkari/stylelint-scss#388]
The `dollar-variable-no-missing-interpolation` rule doesn't work with custom properties

This code snippet does not generate a warning:
```scss
.class {
  $var: 2rem;
  --gap: $var;

  display: grid;
  gap: var(--gap);
}
```

**Comment by kristerkari** [comment]
Thanks! I need to have a look at this one.
